docs/gaia/networks.md

```diff
@@ -1,9 +1,53 @@
 # Networks
 
-There are a variety of ways to setup either local or remote networks with automation, detailed below.
-All the required files are found in the [networks directory](https://github.com/cosmos/cosmos-sdk/tree/develop/networks) and additionally the `local` or `remote` sub-directories.
+This document describes 3 ways to setup a network of `gaiad` nodes, each with an increasing level of complexity:
 
-## Local Testnet
+1. Single-node, local, manual testnet
+2. Multi-node, local, automated testnet
+3. Multi-node, remote, automated testnet
+
+Supporting code can be found in the [networks directory](https://github.com/cosmos/cosmos-sdk/tree/develop/networks) and additionally the `local` or `remote` sub-directories.
+
+> NOTE: The `remote` network bootstrapping may be out of sync with the latest releases and is not to be relied upon.
+
+## Single-node, local, manual testnet
+
+This guide helps you create a single validator node that runs a network locally for testing and other development related uses.
+
+### Requirements
+
+- [Install gaia](./installation.md)
+
+### Create genesis file and start the network
+
+```bash
+# You can run all of these commands from your home directory
+cd $HOME
+
+# Initialize the genesis.json file that will help you to bootstrap the network
+gaiad init --chain-id testing --moniker testing
+
+# Create a key to hold your validator account
+gaiacli keys add validator
+
+# Add that key into the genesis.app_state.accounts array in the genesis file
+# NOTE: this command lets you set the number of coins. Make sure this account has some coins
+# with the genesis.app_state.stake.params.bond_denom denom, the default is STAKE
+gaiad add-genesis-account $(gaiacli keys show validator -o json | jq -r ".address") 1000STAKE,1000validatorToken
```

Comment:
We should note `jq` as a dependency 👍 (<3 `jq` btw).

Reply:
Good call. `jq` is my fave too 😻